arch/x86/kvm/pkvm.c

```diff
@@ -81,3 +82,11 @@ void __init pkvm_reserve(void)
 	kvm_info("Reserved %lld MiB at 0x%llx for pkvm\n", pkvm_mem_size >> 20,
 		 pkvm_mem_base);
 }
+
+void pkvm_init_debugfs(void)
```

Comment:
Maybe let's add the debugfs stuff to a separate file pkvm_debugfs.c like in v6.12? It is kind of an isolated functionality... And for instance, KVM also has a separate file debugfs.c